webpack.config.js

```diff
@@ -10,10 +10,10 @@ const path = require('path');
 const rollbarSnippetPath = './node_modules/rollbar/dist/rollbar.snippet.js';
 const rollbarSnippet = fs.readFileSync(path.join(__dirname, rollbarSnippetPath), { encoding: 'utf8' }).trim();
 
-const patchRete = require('./scripts/patch-rete');
-const reteRelPath = 'node_modules/rete/build/rete.esm.js';
-const retePath = path.join(__dirname, reteRelPath);
-patchRete(retePath);
+// const patchRete = require('./scripts/patch-rete');
+// const reteRelPath = 'node_modules/rete/build/rete.esm.js';
+// const retePath = path.join(__dirname, reteRelPath);
+// patchRete(retePath);
```

Comment:
Are you leaving this in in case we need to patch again?

Reply:
Yes -- It seems like this could easily come up again and having the mechanism available would be easier than trying to reconstruct it.